docs/doc-guide.md

```diff
@@ -24,14 +30,14 @@ GitHub already offers a great guide on how to do this but basically what you nee
 
 ## Step 2: Make the changes in your repository
 
-Now that you have enabled the GitHub page you can start making changes and it's pretty straightforward. You can just make changes to the markdown files inside the *docs* folder or create new ones if are needed. 
+Now that you have enabled the GitHub page you can start making changes and it's pretty straightforward. You can just make changes to the markdown files inside the *docs* folder or create new ones if are needed.
 If you are making a new page make sure that it is linked to another so that the viewers will be able to reach it. Linking the markdown files it's pretty simple and you can find a lot of examples in our repo (you can also read more [here](https://www.codecademy.com/resources/docs/markdown/links)).
 
 Once you've finished making all the changes you can create a commit to your fork and in a few seconds the page should update with all the changes. At this point you should verify that everything looks great and that there are no issue reaching the page.
 
 ## Step 3: Create a PR
 
-If everything went to plan now you should be able to create a PR explaining the changes that you made, in order to facilitate the review process make sure to explain what you have done and add useful screenshots (if you prefer you can just share the link *{your-username}.github.io/sossoldi/*). 
+If everything went to plan now you should be able to create a PR explaining the changes that you made, in order to facilitate the review process make sure to explain what you have done and add useful screenshots (if you prefer you can just share the link *{your-username}.github.io/sossoldi/*).
 After the review process your PR will be merged and your changes should be online!
 
 *Don't know how to open a PR? Don't worry, you can read the [guide](PR-guide.md)*
```

Comment:
This link doesn't work; I believe it needs to be modified to `PR-guide.html`
```suggestion
*Don't know how to open a PR? Don't worry, you can read the [guide](PR-guide.html)*
```

Reply:
Thanks, I'm trying to fix it now. Unfortunately I have to make a new commit every time just to see if it works

Reply:
@theperu Just do a "squash and merge" when you're done to avoid merging dozen commits